src/guides/v2.3/ui_comp_guide/concepts/ui_comp_template_literals.md

```diff
@@ -40,6 +40,23 @@ As a result, a template literal used in the value of one the objects listed abov
 
 Perhaps the most important part of template literals in Magento is the `$` object that can be used inside expressions. (Remember an expression is anything within `${ }`.) The `$` provides access to the `this` context in the JavaScript class where the template literals are. To take it a step further, `this` (and the related `$`) is the KnockoutJS context for the template that can be bound to the UI Component. This object should not be confused with the `$` that marks the beginning of an expression. The `$` object can only appear inside of an expression. Here is an example: `${ $.submitUrl }`: the `$` references the current KnockoutJS context, and `.submitUrl` will return the `provider` property from that object.
 
+### The `ignoreTmpls` property
+
+The `ignoreTmpls` property is an object that allows preventing template processing for selected properties.
+
+```javascript
+return Element.extend({
+    defaults: {
+        value: 'some component value',
+        property: '${ $.value }', // equal to 'some component value' after component initialization
+        propertyIgnoreTempls: '${ $.value }', // equal to '${ $.value }' after component initialization
```

Comment:
This part might be confusing. We may either remove `property` or add an extra comment for `propertyIgnoreTempls` that  `propertyIgnoreTempls` is the property for which template literals will be ignored. Also, we may add an extra comment to `ignoreTmpls` that this section contains list of properties with template literals ignored.

Reply:
Great. Thank you!